x-pack/packages/kbn-elastic-assistant/impl/assistant/translations.ts

```diff
@@ -80,3 +80,24 @@ export const API_ERROR = i18n.translate('xpack.elasticAssistant.assistant.apiErr
   defaultMessage:
     'An error occurred sending your message. If the problem persists, please test the connector configuration.',
 });
+
+export const TOOLTIP_TITLE = i18n.translate(
+  'xpack.elasticAssistant.assistant.technicalPreview.tooltipTitle',
+  {
+    defaultMessage: 'Beta',
+  }
+);
+
+export const TOOLTIP_ARIA_LABEL = i18n.translate(
+  'xpack.elasticAssistant.documentationLinks.ariaLabel',
+  {
+    defaultMessage: 'Click to open Elastic Assistant documentation in a new tab',
+  }
+);
+
+export const DOCUMENTATION = i18n.translate(
+  'xpack.elasticAssistant.documentationLinks.documentation',
+  {
+    defaultMessage: 'Documentation',
```

Comment:
Consider a lowercase `documentation` for consistently with the help links in the screenshot below:

![documentation](https://github.com/elastic/kibana/assets/4459398/38a114d0-ed0c-40cc-bf5f-dd94547bbfb7)